webapp/channels/src/components/announcement_bar/version_bar/version_bar.tsx

```diff
@@ -12,67 +12,52 @@ interface Props {
     buildHash?: string;
 }
 
-interface State {
-    buildHashOnAppLoad?: string;
-}
-
-export default class VersionBar extends React.PureComponent <Props, State> {
-    constructor(props: Props) {
-        super(props);
+const VersionBar = ({
+    buildHash,
+}: Props) => {
+    const [buildHashOnAppLoad, setBuildHashOnAppLoad] = useState(buildHash);
 
-        this.state = {
-            buildHashOnAppLoad: props.buildHash,
-        };
-    }
+    const reloadPage = () => {
+        window.location.reload();
+    };
```

Comment:
This doesn't seem to depend on any prop or anything else from inside the component, so  we can move it outside. That way we don't create the function on every render (and ease the GC work) and we don't pass a new function on every render to the child components.